lib/src/main/java/org/esbtools/eventhandler/PollingDocumentEventProcessorRoute.java

```diff
@@ -50,7 +53,7 @@ public PollingDocumentEventProcessorRoute(DocumentEventRepository documentEventR
     @Override
     public void configure() throws Exception {
         from("timer:pollForDocumentEvents?period=" + pollingInterval.toMillis())
-        .routeId("documentEventsProcessor")
+        .routeId("documentEventsProcessor-" + idCounter.getAndIncrement())
```

Comment:
This and below incrementing id to routeId allows a single camel context to have multiple of these routes, which is useful among other things for testing concurrent scenarios.